cmd/zdb/zdb.c

```diff
@@ -6212,10 +6431,11 @@ main(int argc, char **argv)
 			free(checkpoint_target);
 	}
 
-	if (os != NULL)
+	if (os != NULL) {
```

Comment:
As I remember {} are not required if the body is single-line, so why did you change this condition-block?

Reply:
For consistency with other if() else blocks in the code; see 20 or so lines above, when we have an if () else if() else block where all three clauses are single lines but we use braces anyway.  This change isn't mandatory, but I changed it at an earlier point in the code when this logic got tweaked slightly and didn't remove the change. If it's objectionable, I can revert it.